sql/src/main/java/io/crate/expression/reference/sys/shard/ShardRowContext.java

```diff
@@ -56,12 +57,12 @@
 
     public ShardRowContext(IndexShard indexShard, ClusterService clusterService) {
         this(indexShard, null, clusterService, Suppliers.memoizeWithExpiration(() -> {
-            StoreStats storeStats = indexShard.storeStats();
-            if (storeStats == null) {
-                // will return null if already closed due to shard deletion
```

Comment:
did this behaviour changed recently which would be maybe worth mentioning at the changes entry? (like issue only exists at >= version)

Reply:
Hm, just checked, looks like it's at least this way since 3.2, not sure when exactly it changed.